python/dgl/graphbolt/item_sampler.py

```diff
@@ -756,6 +582,18 @@ def __init__(
             )
         self._world_size = dist.get_world_size()
         self._rank = dist.get_rank()
+        device = (
```

Comment:
how about wrapping this logic into a method as it's dedicated for seed initialization.